classes/item.rb

```diff
@@ -13,4 +13,17 @@ def initialize(publish_date)
     @publish_date = publish_date
     @archived = false
   end
+
+  def move_to_archived?
```

Comment:
- Method name should be `move_to_archive` NOT  `move_to_archived?`

Reply:
Thanks for reporting!

I renamed the method.